pages/visualizations/overview/issue_staleness.py

```diff
@@ -0,0 +1,289 @@
+                                        ),
+                                    ]
+                                ),
+                                dbc.Col(
+                                    dbc.Button(
+                                        "About Graph",
+                                        id="overview-popover-target-is",
+                                        color="secondary",
+                                        size="sm",
+                                    ),
+                                    width="auto",
+                                    style={"padding-top": ".5em"},
+                                ),
+                            ],
+                            align="center",
+                        ),
+                        dbc.Row(
+                            [
+                                dbc.Label(
+                                    "Days Until Staling:",
+                                    html_for="i_staling_days",
+                                    width={"size": "auto"},
+                                    style={"font-weight": "bold"},
+                                ),
+                                dbc.Col(
+                                    dbc.Input(
+                                        id="i_staling_days",
+                                        type="number",
+                                        min=1,
+                                        max=120,
+                                        step=1,
+                                        value=7,
+                                    ),
+                                    className="me-2",
+                                    width=2,
+                                ),
+                                dbc.Label(
+                                    "Days Until Stale:",
+                                    html_for="i_stale_days",
+                                    width={"size": "auto"},
+                                    style={"font-weight": "bold"},
+                                ),
+                                dbc.Col(
+                                    dbc.Input(
+                                        id="i_stale_days",
+                                        type="number",
+                                        min=1,
+                                        max=120,
+                                        step=1,
+                                        value=30,
+                                    ),
+                                    className="me-2",
+                                    width=2,
+                                ),
+                            ],
+                            align="center",
+                        ),
+                        dbc.Alert(
+                            children="Please ensure that 'Days Until Staling' is less than 'Days Until Stale'",
+                            id="i_staling_stale_check_alert",
```

Comment:
id seems to be misspelled- confirm?

Reply:
Mispelled how so? Looks good to me but I might be missing something

Reply:
the first word in the id is 'i' instead of what I would assume, 'is'

Reply:
the i in the front is to distinguish between the pr one. If the id is the same it will cause issues

Reply:
oh 'i' for 'issue' not 'is'. could you please make that 'issue' and 'pr' then for clarity across the two alerts? that's my only comment on quality.

Reply:
fixed!